homeassistant/components/overkiz/climate_entities/atlantic_pass_apc_heating_zone.py

```diff
@@ -26,19 +26,24 @@
     OverkizCommandParam.MANU: HVACMode.HEAT,
     OverkizCommandParam.HEATING: HVACMode.HEAT,
     OverkizCommandParam.STOP: HVACMode.OFF,
+    OverkizCommandParam.EXTERNAL_SCHEDULING: HVACMode.AUTO,
     OverkizCommandParam.INTERNAL_SCHEDULING: HVACMode.AUTO,
     OverkizCommandParam.COMFORT: HVACMode.HEAT,
 }
 
 HVAC_MODE_TO_OVERKIZ = {v: k for k, v in OVERKIZ_TO_HVAC_MODE.items()}
 
+PRESET_EXTERNAL = "external"
+PRESET_FROSTPROTECTION = "frostprotection"
```

Comment:
```suggestion
PRESET_FROSTPROTECTION = "frost_protection"
```
`overkiz/climate_entities/atlantic_electrical_heater_with_adjustable_temperature_setpoint.py` is telling `frost_protection`, not `frost_protection`.

Which is the good ?

Reply:
Preset could be whatever we want, it's what is shown to the user.
Somfy constant is litterally "frostprotection", So I would say that frostprotection is correct.  
I don't think updating the other entity will be a good idea...

So should I update this line of code or keep 2 differents versions for 2 differents devices?

Reply:
`frostprotection` is two words and thus should have an underscore. Please adjust this accordingly.